Userland/Libraries/LibWeb/Layout/BlockContainer.h

```diff
@@ -22,8 +26,13 @@ class BlockContainer : public Box {
 
     Painting::PaintableWithLines const* paintable_with_lines() const;
 
+    void store_layout_inside_run_info(LayoutState* layoutState, AvailableSpace const availableSpace);
+
     virtual JS::GCPtr<Painting::Paintable> create_paintable() const override;
 
+    // Save info for layout invalidation input box fast path
+    AvailableSpace m_run_available_space = AvailableSpace(AvailableSize(AvailableSize::Type::Definite, CSSPixels(0)), AvailableSize(AvailableSize::Type::Definite, CSSPixels(0)));
```

Comment:
Use `AvailableSize::make_definite(CSSPixels(0))` for these. (You might even be OK just putting `0` instead of `CSSPixels(0)`)